src/torchrunx/logging_utils.py

```diff
@@ -1,18 +1,19 @@
 from __future__ import annotations
 
+import datetime
 import logging
 import pickle
 import struct
 from io import StringIO, TextIOWrapper
 from logging import Handler, Logger
 from logging.handlers import SocketHandler
-from socketserver import StreamRequestHandler, TCPServer
+from socketserver import StreamRequestHandler, ThreadingTCPServer
 
 
 def log_records_to_socket(
     logger: Logger, hostname: str, worker_rank: int | None, logger_hostname: str, logger_port: int
 ):
-    logger.setLevel(logging.NOTSET)
+    logger.setLevel(logging.DEBUG)
```

Comment:
Why `DEBUG` rather than `NOTSET`?

According to this
https://docs.python.org/3/library/logging.html#logging.Logger.setLevel

`NOTSET` in the root logger means all messages will be processed. I think that's what we want?